tests/plot_sync/test_receiver.py

```diff
@@ -259,7 +277,23 @@ async def test_to_dict(counts_only: bool) -> None:
     assert get_list_or_len(sync_steps[State.keys_missing].args[0], counts_only) == plot_sync_dict_3["no_key_filenames"]
     assert get_list_or_len(sync_steps[State.duplicates].args[0], counts_only) == plot_sync_dict_3["duplicates"]
 
+    assert plot_sync_dict_3["total_plot_size"] == sum(plot.file_size for plot in receiver.plots().values())
     assert plot_sync_dict_3["last_sync_time"] > 0
+    assert plot_sync_dict_3["syncing"] is None
+
+    # Trigger a repeated plot sync
+    await receiver.sync_started(
+        PlotSyncStart(
+            PlotSyncIdentifier(uint64(time.time()), uint64(receiver.last_sync().sync_id + 1), uint64(0)),
+            False,
+            receiver.last_sync().sync_id,
+            uint32(1),
+        )
+    )
+    sync_data = receiver.to_dict()["syncing"]
+    assert not sync_data["initial"]
+    assert sync_data["plot_files_processed"] == 0
+    assert sync_data["plot_files_total"] == 1
```

Comment:
See above.

Reply:
Thanks, pretty sure you complained about something similar already at some point but i forgot haha. Applied both, see d0c961e43f954c63785acf6afc89f431ade3f6f4